lib/linux/background-managers/dconf.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const childProcess = require('child_process');
+const util = require('util');
+const {existCommand} = require('../util');
+
+const execFile = util.promisify(childProcess.execFile);
+
+exports.available = function () {
+	return existCommand('dconf');
+};
+
+exports.set = async function (imagePath) {
+	await execFile('dconf', ['write', '/org/mate/desktop/background/picture-filename', '"' + imagePath + '"']);
+};
+
+exports.get = async function () {
+	try {
+		const {stdout} = await execFile('dconf', ['read', '/org/mate/desktop/background/picture-filename']);
+
+		return stdout.trim().slice(1, -1);
+	} catch (_) {}
```

Comment:
Why are you ignoring the error?

Reply:
i'm not a nodejs coder. If my memories are good, I write it to prevent the resolutions of promises to fail in the file index.js. I have some difficulty about the understanding of `await` and `async`.

Maybe the best solution is to add `try{ }catch(){ }` on the [line 43](https://github.com/deather/wallpaper/blob/22b01cfa30fa683f93015d48acef600de49c68e8/lib/linux/index.js#L43). But I don't know where to write the next lines.

I hope you can understand my explanations.

Reply:
Actually, would be better to just change line 31 to `const imagePath = await app.get().catch(() => {});`